binnacle/lib/providers/bluetooth.dart

```diff
@@ -85,24 +81,43 @@ class BluetoothManager {
     deviceConnection = _flutterBlue
         .connect(device, timeout: const Duration(seconds: 4))
```

Comment:
Is there a reason for this specific timeout value? We might want to shorten or lengthen it and see if it has any effect on the sensor package's reliability.